img/alt-tags

```diff
@@ -0,0 +1,5 @@
+/* Proposed Photo Alt Tag */
```

Comment:
I believe these are great as a start, and love that the alt tags are being utilized and worked into the image. To make the image even easier, maybe rename the files to be less length, and even reduce the file size as each image appears to be quite large.